canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -265,6 +280,20 @@ class DropDown extends React.Component {
 					disabled={this.props.state === STATES.DISABLED || this.disableEmptyListDropdown}
 					type="default"
 					items={dropDown.options}
+					itemToString={(item) => (item ? (
+						<div className="item-name">
+							<div>{ item.label }</div>
+							<div className="custom-icon-label">{this.testIconCallBack()}</div>
+						</div>
+					) : ""
+					)}
+					itemToElement={(item) => (item ? (
+						<div data-id={`properties-${this.props.propertyId}`} className="item-name">
```

Comment:
data-id isn't usually set on each specific value.  That seems wrong.  `item-name` also seems too generic.  We should at least add properties-...